Packs/SumoLogic_Cloud_SIEM/Integrations/SumoLogicCloudSIEM/SumoLogicCloudSIEM.yml

```diff
@@ -58,6 +64,9 @@ script:
       required: true
       default: true
       description: The insight to retrieve details for.
+    - name: record_summary_fields
```

Comment:
for both `record_summary_fields` arguments added, probably better to set the default as the `defaultValue` rather than in the code

Reply:
Done